spi/identity-hub-spi/src/testFixtures/java/org/eclipse/dataspaceconnector/identityhub/junit/testfixtures/VerifiableCredentialTestUtil.java

```diff
@@ -0,0 +1,102 @@
+ *  Contributors:
+ *       Microsoft Corporation - initial API and implementation
+ *
+ */
+
+package org.eclipse.dataspaceconnector.identityhub.junit.testfixtures;
+
+import com.github.javafaker.Faker;
+import com.nimbusds.jose.JOSEException;
+import com.nimbusds.jose.JWSAlgorithm;
+import com.nimbusds.jose.JWSHeader;
+import com.nimbusds.jose.crypto.ECDSASigner;
+import com.nimbusds.jose.jwk.Curve;
+import com.nimbusds.jose.jwk.ECKey;
+import com.nimbusds.jose.jwk.KeyUse;
+import com.nimbusds.jose.jwk.gen.ECKeyGenerator;
+import com.nimbusds.jwt.JWTClaimsSet;
+import com.nimbusds.jwt.SignedJWT;
+import org.eclipse.dataspaceconnector.iam.did.crypto.key.KeyConverter;
+import org.eclipse.dataspaceconnector.iam.did.spi.document.EllipticCurvePublicKey;
+import org.eclipse.dataspaceconnector.iam.did.spi.key.PublicKeyWrapper;
+import org.eclipse.dataspaceconnector.identityhub.credentials.model.VerifiableCredential;
+
+import java.time.Duration;
+import java.time.Instant;
+import java.util.Date;
+import java.util.Map;
+
+/**
+ * Util class to manipulate VerifiableCredentials in tests.
+ */
+public class VerifiableCredentialTestUtil {
+    public static final Date EXP = Date.from(Instant.now().plus(Duration.ofDays(1)));
+    private static final Faker FAKER = new Faker();
+    private static final ECKeyGenerator EC_KEY_GENERATOR = new ECKeyGenerator(Curve.P_256);
+
+    public static VerifiableCredential generateVerifiableCredential() {
+        return VerifiableCredential.Builder.newInstance()
+                .id(FAKER.internet().uuid())
+                .credentialSubject(Map.of(
+                        FAKER.internet().uuid(), FAKER.lorem().word(),
+                        FAKER.internet().uuid(), FAKER.lorem().word()))
+                .build();
+    }
+
+    public static ECKey generateEcKey() {
+        try {
+            return EC_KEY_GENERATOR.keyUse(KeyUse.SIGNATURE).generate();
+        } catch (JOSEException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    public static PublicKeyWrapper toPublicKeyWrapper(ECKey jwk) {
+        var publicKey = new EllipticCurvePublicKey(jwk.getCurve().getName(), jwk.getKeyType().getValue(), jwk.getX().toString(), jwk.getY().toString());
+        return KeyConverter.toPublicKeyWrapper(publicKey, "ec");
+    }
+
+    public static SignedJWT buildSignedJwt(VerifiableCredential credential, String issuer, String subject) throws Exception {
+        var jwk = new ECKeyGenerator(Curve.P_256).keyUse(KeyUse.SIGNATURE).generate();
+        return buildSignedJwt(credential, issuer, subject, jwk);
+    }
+
+    public static SignedJWT buildSignedJwt(VerifiableCredential credential, String issuer, String subject, ECKey jwk) {
+        try {
+            var jwsHeader = new JWSHeader.Builder(JWSAlgorithm.ES256).build();
+            var claims = new JWTClaimsSet.Builder()
+                    .claim("vc", credential)
+                    .issuer(issuer)
+                    .expirationTime(EXP)
```

Comment:
I would remove this claim (`expirationTime`) and use it specifically in dedicated UTs